webapi/src/main/scala/org/knora/webapi/util/PermissionUtilV1.scala

```diff
@@ -316,106 +315,36 @@ object PermissionUtilV1 {
         // Anything with permissions must have an creator and a project.
         val subjectCreator: IRI = assertionMap.getOrElse(OntologyConstants.KnoraBase.AttachedToUser, throw InconsistentTriplestoreDataException(s"Subject $subjectIri has no creator"))
         val subjectProject: IRI = assertionMap.getOrElse(OntologyConstants.KnoraBase.AttachedToProject, throw InconsistentTriplestoreDataException(s"Subject $subjectIri has no project"))
-        val subjectPermissionLiteral: Option[String] = assertionMap.get(OntologyConstants.KnoraBase.HasPermissions)
+        val subjectPermissionLiteral: String = assertionMap.getOrElse(OntologyConstants.KnoraBase.HasPermissions, throw InconsistentTriplestoreDataException(s"Subject $subjectIri has no knora-base:hasPermissions predicate"))
```

Comment:
Is this consistent? I think there are a lot of places in the code where there is no check at all because the existence of a permission is assumed (required by the ontology).

Otherwise all the old `get` (resulting in an option of a permission) should consistently be changed to `getOrElse` to throw an apt error message and not getting a Java-exception because the element was not found.